LukeMason33/Rotten-Tomatillos
# Fix issue of doubling up movies. Put in conditional method in render …

### Is this a fix or a feature?
- This is a fix.


### What is the change/fix?
- After our last merge we were having probems with the exiting conditionals inside of the render method in App.js. We refactored this conditional check into it's own function which we then call inside of the return statement within the render method.


### Where should the reviewer start?
- The reviewer should start in App.js and look at the render method within that component.



### How should this be tested?
- This should be tested by opening up the app and makin sure that: movies are not duplicated, the search function works, the filter dropdown method works, and you can view movie details as well as go back to the main screen.
